acra/configuring-maintaining/storage-and-data-model-implications/_index.md

```diff
@@ -2,3 +2,79 @@
 title: Storage and data model implications
 bookCollapseSection: true
 ---
+
+# Data storage
+
+Based on which Acra components you use, the requirements may differ.
+
+For example, you will most likely want to use AcraServer.
+In this case, you are limited by supported DBMS, which are MySQL, MariaDB, PostgreSQL.
+
+On the other hand, there is AcraTranslator and it does not depend on storage you use.
+It's just a side service that performs crypto operations and you are free to store encrypted data in
+any other database (including embedded ones), in any key/value storage and so on.
+The main requirement for a storage would be support of binary fields (value as array of bytes, blob, etc).
+But even in this case you can, for example, base64 encode encrypted data and store it as a string.
+
+# Type compatibility
+
+When encrypting, Acra takes a bytestring and returns a bytestring.
+So all the data you wanna protect should be transformed into a bytestring:
+1) text strings can be taken "as is"
```

Comment:
actually, it is not true because DBMS that expects strings will validate encoding and will return errors to clients if strings in invalid encoding after encryption. It is supported only if database doesn't validate encoding (MySQL in some cases, but not Postgresql).
imho better to remove this line at all )